src/app/complex/complex-results/complex-navigator/complex-navigator.component.ts

```diff
@@ -0,0 +1,14 @@
+import {Component, Input} from '@angular/core';
+import {ComplexSearchResult} from '../../shared/model/complex-results/complex-search.model';
+import {Interactor} from '../../shared/model/complex-results/interactor.model';
+
+@Component({
+  selector: 'cp-complex-navigator',
+  templateUrl: './complex-navigator.component.html',
+  styleUrls: ['./complex-navigator.component.css']
+})
+export class ComplexNavigatorComponent {
+  @Input() complexSearch: ComplexSearchResult;
+  @Input() interactors: Interactor[];
+  interactorsSorting: string = 'Occurrence';
```

Comment:
When a variable is set and the type is clear, like a string or a number, typescript doesn't need the type and complains about it actually.

Here, because `'Occurrence'` is clearly a string, we don't need `:string`, we can simply do.
```
interactorsSorting = 'Occurrence';
```

It's the same with numbers
```
const a: number = 1; // typescript does not like it
const b = 1; // typescript is happy
```